dimagi_lake/aggregation/aggregation_helpers/agg_flwc_administration.py

```diff
@@ -0,0 +1,138 @@
+from datetime import datetime
+
+from dimagi_lake.aggregation.aggregation_helpers.base_helper import \
+    BaseAggregationHelper
+from dimagi_lake.utils import clean_name
+from spark_session_handler import SPARK
+
+
+class AggFlwcAdministartionHelper(BaseAggregationHelper):
+
+
+    @property
+    def source_tablename(self):
+        return "flwc_management_flwc_administration"
+
+    def aggregate(self):
```

Comment:
I think this class reinforces [this comment](https://github.com/dimagi/dimagi_lake/pull/1/files#r610429784). Seems like the base class shouldn't bother with those other methods called in aggregation (besides `enforce_schema`) unless you intend them to provide an external interface down the line, since this one used none of them.